.k8s/components/strapi.ts

```diff
@@ -129,28 +91,62 @@ const createStrapiComponent = async (
     },
   });
 
-  manifests.push(...strapiManifests);
-  manifests.push(pvc);
-
-  return await manifests;
+  return manifests.concat([persistentVolumeClaim, persistentVolume]);
 };
 
-export default async () => {
-  // generate basic strapi manifests
-  const manifests = await createStrapiComponent("strapi", {
-    config: {
-      subDomainPrefix: "backoffice-",
-      image: getHarborImagePath({ name: "les1000jours-strapi" }),
-      pvcName: "1000jours-strapi-uploads" // persistant storage name
+function azureProjectVolume(name: string, { storage }: { storage: string }) {
```

Comment:
can we put this in `kosko-charts/utils` ?

Reply:
yes